src/features/devices/pages/DevicesPage.tsx

```diff
@@ -17,26 +22,26 @@ export const DevicesPage = () => {
   return (
     <section className="w-full h-full pt-[16px]">
       <div className="mb-[24px]">
-        <ButtonBack size={32}/>
+        <ButtonBack size={32} />
```

Comment:
Check for over-formatting before creating a PR, it makes the review process more difficult. Not this time, but in the future, please check🤔